concent_api/core/tests/test_core_views.py

```diff
@@ -424,3 +470,67 @@ def test_view_receive_out_of_band_return_http_204_if_no_messages_in_database(sel
 
         self.assertEqual(response.status_code, 204)
         self.assertEqual(response.content.decode(), '')
+
+    def test_two_receive_out_of_band_in_row(self):
```

Comment:
I am not sure if this test expectations are correct. It means that each time somebody (for now without key checking) hits `\receive_out_of_band` he will get VerdictReportComputedTask. @kbeker @cameel is that expected behavior ?

Reply:
The test is wrong and needs to be changed. `VerdictReportComputedTask` should be returned only once.

By the way the name of the test function does not follow our convention. The pattern is `def test_<what you're testing>_should_<what you expect it to do>():`